README.md

```diff
@@ -64,11 +64,34 @@ See [CONTRIBUTING.md](./CONTRIBUTING.md)
 
 ### Local development with Kubebuilder
 
-1. Install `pre-commit`: see <https://pre-commit.com/#install>
-1. Install `kubebuilder`: see <https://book.kubebuilder.io/quick-start.html#installation>
-1. Install `ArgoCD Application` API pkg: see `hack/install-argocd-application.sh`
+Just run
 
-### Update ArgoCD Application API package
+```
+bash hack/install-dev-deps.sh
+bash hack/setup-dev.sh [argocd-version] [appset-version]
+```
+
+this will install all the dependencies (`pre-commit`, `kubebuilder`, `argocd`, `kind`) and it will install the correct version of ArgoCD Application API package for you. If you omit `argocd-version` and/or `appset-version` it will default to the latest stable/tested versions of ArgoCD and Appset controller.
+
+After running the script, you will have 3 kind clusters created locally:
```

Comment:
Should we differentiate between instructions to run the controller locally with the kubebuilder api server and etcd, and a more e2e environment with a real kubernetes cluster, argocd and applicationset?

For developing the controller should be enough what we get with kubebuilder.

Reply:
I think I've addressed this now. Let me know what you think.